spec/requests/users_spec.rb

```diff
@@ -71,12 +92,49 @@
   end
 
   it 'should return 400 if user does not exist' do
-    expected_response = { 'error' => 'User Not Found' }
-
     expect do
       delete '/users/420'
     end.to_not change { User.count }
-    expect(json_body).to eq(expected_response)
+    expect(response.status).to eq(400)
+  end
+end
+
+RSpec.describe 'POST /users/:id/favorites' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.